clean.sh

```diff
@@ -1,9 +1,11 @@
+#!/usr/bin/env bash
 # Copyright (c) 2018 Dan Petre
-
 # The MIT License (MIT)
-
 rm -rf build/
 rm -rf dist/
 rm -rf *.egg-info/
 rm -rf blocksWorld/*.pyc
 rm -rf blocksWorld/__pycache__
+rm -rf ./.pytest_cache/
+rm -rf ./.eggs/
+rm -rf blocksWorld.egg-info
```

Comment:
yay, more things to ignore - that's a good thing.

Minor comment: leave an empty line at the end of each file. For some reason git likes that.